code/obj/item/clothing/suits.dm

```diff
@@ -1118,6 +1118,11 @@
 			icon_state = "syndie_specialist-infiltrator"
 			item_state = "syndie_specialist-infiltrator"
 
+		setupProperties()
+			..()
+			setProperty("space_movespeed", -0.25)
```

Comment:
This only affects movement speed in space I think? Rather than "movespeed" for general movement.
Unless that's the intention (albeit I don't see mention of space in the description).

Reply:
I thought that too but it apparently doesn't work that way. 
The space movespeed property effects your standard move speed, but you move at full speed when in space. The knight armour uses the same property.

Reply:
*scream